site/src/components/Footer.tsx

```diff
@@ -87,38 +83,67 @@ const SOCIALS: { name: string; icon: ReactNode; href: string }[] = [
     icon: <Icon className="fab fa-discord" />,
     href: "https://discord.gg/PefPteFe5j",
   },
+  {
+    name: "GitHub",
+    icon: <Icon className="fab fa-github" />,
+    href: "https://github.com/blockprotocol",
+  },
 ];
 
 const Socials = (
-  <Box mt={3} display="flex">
-    {SOCIALS.map(({ href, icon }) => (
-      <Link
-        href={href}
-        key={href}
-        sx={{
-          "&:not(:first-child)": {
-            marginLeft: 2.5,
-          },
-          ":hover": {
-            svg: {
-              color: (theme) => theme.palette.gray[30],
+  <Box
+    mt={3}
+    display="flex"
+    sx={{
+      flexDirection: {
+        xs: "column",
+        md: "row",
+      },
+      alignItems: { xs: "left", md: "center" },
+    }}
+  >
+    <Box marginBottom="1rem">
+      {SOCIALS.map(({ href, icon }) => (
+        <Link
+          href={href}
+          key={href}
+          sx={{
+            marginRight: 2.5,
+            ":hover": {
+              svg: {
+                color: (theme) => theme.palette.gray[30],
+              },
+            },
+            ":active": {
+              svg: {
+                color: (theme) => theme.palette.common.white,
+              },
             },
-          },
-          ":active": {
             svg: {
-              color: (theme) => theme.palette.common.white,
+              transition: (theme) =>
+                theme.transitions.create("color", { duration: 150 }),
+              color: (theme) => theme.palette.gray[50],
             },
-          },
-          svg: {
-            transition: (theme) =>
-              theme.transitions.create("color", { duration: 150 }),
-            color: (theme) => theme.palette.gray[50],
-          },
+          }}
+        >
+          {icon}
+        </Link>
+      ))}
+    </Box>
+    <a href="https://github.com/blockprotocol/blockprotocol">
+      <Button
+        variant="primary"
+        size="small"
+        sx={{
+          marginBottom: 2.5,
+          backgroundColor: (theme) => theme.palette.gray[70],
+          color: (theme) => theme.palette.gray[20],
         }}
+        startIcon={<Icon className="fa fa-star" />}
       >
-        {icon}
-      </Link>
-    ))}
+        Star us on Github
+      </Button>{" "}
```

Comment:
I think this trailing space is unintentional, likely introduced whilst running `prettier` or `lint:fix`:
```suggestion
      </Button>
```

Reply:
Have made this change in https://github.com/blockprotocol/blockprotocol/pull/75/commits/ae821f0ba94307366141bb6428de3c7c7abe9030